source/particle/world.cc

```diff
@@ -85,19 +85,26 @@ namespace aspect
     std::string
     World<dim>::generate_output() const
     {
+      std::vector<std::string> filenames;
```

Comment:
What do you use this vector for? Do you intend to return all file names? But what to do with them?